### DISCUSSION TITLE
How do you define a scene in the scene manager tool?

### DISCUSSION THREAD

**Discussion by altinaaaaaaa** [maktoobgar/scene_manager#18] [Q&A] [answered]
I just added the plugin and my open scenes were listed under "Uncategorized" in the Scene Manager panel. I copied the variables from the demo script, and in my scene I have this line

```
SceneManager.change_scene("res://scenes/levels/battle.tscn", fade_out_options, fade_in_options, general_options)
```

However, once I try to run my scene I get the error `Assertion failed: Scene Manager Error: loaded scene is not defined in scene manager tool.`

How do you define a scene in the scene manager tool?

**Comment by maktoobgar** [comment, marked as answer]
Hello,
First, know that I created the whole plugin to actually get rid of giving that whole big address to change a simple scene.
What you pass as the first parameter of the `SceneManager.change_scene` function is a key which represents a scene you define in the scene manager UI tool on the right side of the godot editor.

And second, it seems that I have to make that error message more informing because you are not the first person who asks about that message and that message means that this scene which you currently are running was not found in the godot scene manager plugin scenes.
What you need to do to get rid of that error message is to go to scene manager UI tool on the right side and click on refresh and then save and you should not see that message any more.
